### PR TITLE
Add vaadin-custom-field to range date picker demo [skip ci]

### LINKED ISSUE #630: Update range demo to use vaadin-custom-field
A user has asked me about the range date picker in Slack, so I have recommended the `vaadin-custom-field`, but then he pointed out that there is a different example here.

We should consider using `vaadin-custom-field` in demos until we introduce range support.

### PR DESCRIPTION
**Pull request by niiyeboah**
Fixes #630

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-date-picker/631)

### CODE REVIEW

File: demo/date-picker-basic-demos.html [hunk 1]
```diff
@@ -47,9 +47,10 @@ <h3>Date range</h3>
     <p>Use two date pickers to create a date range selector.</p>
```
Comment (web-padawan) [inline demo/date-picker-basic-demos.html]: Let's update the description to mention the component used in demo:

```html
Wrap two date pickers in <a href="https://vaadin.com/components/vaadin-custom-field"><code>&lt;vaadin-custom-field&gt;</code></a> to create a date range selector.
```

File: demo/date-picker-basic-demos.html [hunk 2]
```diff
@@ -47,9 +47,10 @@ <h3>Date range</h3>
...
     <vaadin-demo-snippet id="date-picker-basic-demos-date-range" when-defined="vaadin-date-picker">
       <template preserve-content>
-        <vaadin-date-picker id="start" label="Start"></vaadin-date-picker>
-        <vaadin-date-picker id="end" label="End"></vaadin-date-picker>
-
+        <vaadin-custom-field label="Date range">
```
Comment (web-padawan) [inline demo/date-picker-basic-demos.html]: Let's update the demo to output the custom field value below, same as in the [combo-box demo](https://vaadin.com/components/vaadin-combo-box/html-examples/combo-box-basic-demos#allow-custom-values).

Comment (niiyeboah) [reply demo/date-picker-basic-demos.html]: Done

File: demo/date-picker-basic-demos.html (hunk 1 above)
Comment (niiyeboah) [reply demo/date-picker-basic-demos.html]: Done

**Pull request merged**